stock_by_warehouse/static/src/js/widget.js

```diff
@@ -6,27 +6,34 @@ var Widget = require('web.Widget');
 var form_common = require('web.AbstractField');
 var formats = require('web.field_utils');
 var fieldRegistry = require('web.field_registry');
+var {EventDispatcherMixin} = require('web.mixins');
```

Comment:
Why the braces?

Reply:
To only obtain the class `EventDispatcherMixin` from the file `web.mixins`

Reply:
Ok, resolving.